mobile-app/app/screens/AppNavigator/screens/Portfolio/components/TokenAmountTextV2.tsx

```diff
@@ -29,7 +29,7 @@ export function TokenAmountTextV2 ({
       renderText={(value) =>
         <View style={tailwind('flex flex-1')}>
           <BalanceTextV2
-            containerStyle={[tailwind('justify-end text-right'), { marginBottom: Platform.OS === 'android' ? 2 : 3 }]}
+            containerStyle={[tailwind('text-sm font-semibold-v2 justify-end text-right mb-1')]}
```

Comment:
Looks fine on ios, but not on android:
![image](https://user-images.githubusercontent.com/40191153/183328981-92858351-2174-420f-b4a4-b62ee90e9251.png)